index.Rmd

```diff
@@ -126,7 +126,7 @@ Show tables, plots, etc. and describe them.
 
 ```
 
-# Conclusions
+# Conclusions 
```

Comment:
Tie up your results into a conclusion section here ie. what are the key takeaways of your project?